image_registry.yaml

```diff
@@ -0,0 +1,294 @@
+  tags:
+    - v1.15.1
+    - v1.15.4
+    - v1.16.4
+cert_manager_cainjector:
+  default_tag: v1.16.4
+  name: quay.io/jetstack/cert-manager-cainjector
+  tags:
+    - v1.15.1
+    - v1.15.4
+    - v1.16.4
+cert_manager_controller:
+  default_tag: v1.16.4
+  name: quay.io/jetstack/cert-manager-controller
+  tags:
+    - v1.15.1
+    - v1.15.4
+    - v1.16.4
+cert_manager_ctl:
+  default_tag: v1.16.4
+  name: quay.io/jetstack/cert-manager-ctl
+  tags:
+    - v1.15.1
+    - v1.15.4
+    - v1.16.4
+cert_manager_webhook:
+  default_tag: v1.16.4
+  name: quay.io/jetstack/cert-manager-webhook
+  tags:
+    - v1.15.1
+    - v1.15.4
+    - v1.16.4
+cloudnative_pg:
+  default_tag: 1.25.1
+  name: ghcr.io/cloudnative-pg/cloudnative-pg
+  tags:
+    - 1.23.2
+    - 1.24.0
+    - 1.24.1
+    - 1.25.1
+ferretdb:
+  default_tag: 1.24.1
+  name: ghcr.io/ferretdb/ferretdb
+  tags:
+    - 1.23.0
+    - 1.24.0
+    - 1.24.1
+forgejo:
+  default_tag: 1.21.11-2
+  name: codeberg.org/forgejo/forgejo
+  tags:
+    - 1.21.11-2
+frrouting_frr:
+  default_tag: 9.1.0
+  name: quay.io/frrouting/frr
+  tags:
+    - 9.1.0
+grafana:
+  default_tag: 11.3.1
+  name: grafana/grafana
+  tags:
+    - 10.4.5
+    - 11.2.0
+    - 11.3.1
+istio_pilot:
+  default_tag: 1.24.5-distroless
+  name: docker.io/istio/pilot
+  tags:
+    - 1.22.3-distroless
+    - 1.23.1-distroless
+    - 1.23.2-distroless
+    - 1.23.3-distroless
+    - 1.24.5-distroless
+istio_proxy:
+  default_tag: 1.24.5-distroless
+  name: docker.io/istio/proxyv2
+  tags:
+    - 1.22.3-distroless
+    - 1.23.1-distroless
+    - 1.23.2-distroless
+    - 1.23.3-distroless
+    - 1.24.5-distroless
+jupyter_datascience_lab:
+  default_tag: lab-4.4.0
+  name: quay.io/jupyter/datascience-notebook
+  tags:
+    - lab-4.4.0
+karpenter:
+  default_tag: 1.2.0
+  name: public.ecr.aws/karpenter/controller
+  tags:
+    - 0.37.0
+    - 1.0.8
+    - 1.2.0
+keycloak:
+  default_tag: 26.2.0
+  name: quay.io/keycloak/keycloak
+  tags:
+    - 25.0.2
+    - 25.0.6
+    - 26.2.0
+# Be sure to check:
+# https://kiali.io/docs/installation/installation-guide/prerequisites/#version-compatibility
+kiali:
+  default_tag: v2.8.0
+  name: quay.io/kiali/kiali
+  tags:
+    - v1.87.0
+    - v1.89.7
+    - v2.8.0
+kiwigrid_sidecar:
+  default_tag: 1.30.3
+  name: quay.io/kiwigrid/k8s-sidecar
+  tags:
+    - 1.27.4
+    - 1.30.3
+knative_istio_controller:
+  default_tag: &knative_istio_default_tag v1.17.0
+  name: gcr.io/knative-releases/knative.dev/net-istio/cmd/controller
+  tags: &knative_istio_allowed_tags
+    - v1.15.1
+    - v1.17.0
+knative_istio_webhook:
+  default_tag: *knative_istio_default_tag
+  name: gcr.io/knative-releases/knative.dev/net-istio/cmd/webhook
+  tags: *knative_istio_allowed_tags
+knative_serving_activator:
+  default_tag: &knative_default_tag v1.17.0
+  name: gcr.io/knative-releases/knative.dev/serving/cmd/activator
+  tags: &knative_allowed_tags
+    - v1.15.1
+    - v1.15.2
+    - v1.17.0
```

Comment:
Here's an example of the anchor and alias setup that aims to replicate how we did it with `@registry`